sql/log_event.cc

```diff
@@ -5534,6 +5534,9 @@ int Query_log_event::do_apply_event(rpl_group_info *rgi,
   */
   if (is_trans_keyword() || rpl_filter->db_ok(thd->db.str))
   {
+#ifdef WITH_WSREP
+    if (!(wsrep_on(thd) && wsrep_thd_is_applying(thd)))
+#endif
     thd->set_time(when, when_sec_part);
```

Comment:
if thd is applying it certainly has wsrep_on as well, so this condition could drop the wsrep_on(thd) requirement

Reply:
Done.